packages/DApp/src/hooks/useFeaturedVotes.ts

```diff
@@ -56,7 +56,7 @@ export function useFeaturedVotes() {
 
     loadFeatureVotes()
 
-    const task = setInterval(loadFeatureVotes, 10000)
+    const task = setInterval(loadFeatureVotes, 5000)
```

Comment:
why decreasing the interval? given the `history` statements (https://github.com/status-im/community-dapp/pull/88/files#r1377631506) aren't those enough of a workaround already?

Reply:
It's different kind of story. This updates votes (and everything related) in UI from waku. So if someone votes during my visit at this page, it updates votes for me as well.

Reply:
Right, but if I understand the workaround correctly, it will update the votes for you after you've interacted with the contract. Letting you know about the updates done or new action to be taken then.

The reason I pointed this line out is that I'm convinced it puts even more stress on the node. Instead of changing the Waku protocol if optimizations in this case are to be made.

> consider replacing setInterval(get, 10000) for Filter protocol and use Store only on initial load – https://github.com/status-im/status-desktop/issues/7534#issuecomment-1330515894

Reply:
reverted https://github.com/status-im/community-dapp/pull/88/commits/ff3a9468526c5598655df46d43a14632366b0bd1